pkgs/applications/misc/tvbrowser/default.nix

```diff
@@ -40,27 +39,28 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/share/${pname}
-    cp -R runtime/tvbrowser_linux/* $out/share/${pname}
+    mkdir -p $out/share/tvbrowser
```

Comment:
@aaronArinder

Reply:
howdy, i think you've the wrong aaron?

Reply:
Wrong mark, sorry. I forgot the Y in @Yarny0.

Reply:
Is there a problem with this change?

Reply:
Nope, it is OK.

The only problem is now ofborg is not evaluating...